internal/proxy/proxy.go

```diff
@@ -100,6 +103,9 @@ func (c *Config) DialerOptions(l alloydb.Logger) ([]alloydbconn.Option, error) {
 	opts := []alloydbconn.Option{
 		alloydbconn.WithUserAgent(c.UserAgent),
 	}
+
+	l.Infof("Using API Endpoint %v", c.APIEndpointURL)
```

Comment:
We can remove this and move the option below back into the slice above as you initially had.